cross_language_tests/nacler_cross_test.go

```diff
@@ -50,6 +50,9 @@ func TestNaclerRuby(t *testing.T) {
 	bobRubyKey, err := ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
 	require.NoError(t, err)
 
+	malloryKey, err := ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
+	require.NoError(t, err)
+
 	naclerTests := []struct {
```

Comment:
Wonder if we should add a TPM keyer to these tests? There is a TPM keyer test in `pkg/nacler` that test the tpm keys against the go local keys. This tests the go local keys against ruby local keys, by transitive property, tpm keys will work against ruby local keys?

Reply:
I added the tpm to the cross tests ... it was painful due to the ruby install action adding an incompatible version of gcc to the path, but got it figured out.